### ISSUE TITLE
NUnit2057 when asserting thrown exception

### ISSUE THREAD

**Issue by appel1** [nunit/nunit.analyzers#951]
This triggers NUnit2057. 

```csharp
Assert.That(() => MethodThatThrows(), Throws.Exception);
```

The code fix replaces it with the following that makes the test fail. 
```csharp
Assert.That(MethodThatThrows(), Throws.Exception);
```

Is there a better way to assert a thrown exception? Or should the rule be aware of Throws? 

Using NUnit 4.4.0 with NUnit.Analyzers 4.11.1.

**Comment by mikkelbu** [comment]
Thanks for the report @appel1 - This should have been fixed been fixed in Release 4.11.2. Which version do you use?

**Comment by manfred-brands** [comment]
This has been fixes in version 4.11.2
